<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>385</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>385</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_consoleSelect">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GBA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NDS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New 3DS/2DS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Old 3DS/2DS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60 FPS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_IntroTimer">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="value">
     <number>35000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_EncounterAdv">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="value">
     <number>700</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Intro Timer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encounter Advances</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>229</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>289</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Advances Time</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_TotalTimeResult">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>249</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_TotalAdvancesResult">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>309</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>-9</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="pushButton_calculate_adv2ms">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_calculate_ms2adv">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_MStoAdv_Input">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Adv2MS_Input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_MStoAdv_Result">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Adv2MS_Result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Advances to MS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MS to Advances</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fps_display">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>59.7275</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>171</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>201</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_calculate">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
